backend/src/modules/users/users.dto.ts

```diff
@@ -1,10 +1,13 @@
 import {
   IsEmail,
+  IsEnum,
   IsNotEmpty,
+  IsOptional,
   IsString,
   Length,
   Matches,
 } from 'class-validator';
+import { UserAuthProvider } from 'src/enums/user.auth.provider';
 
 export class CreateUserDto {
```

Comment:
i think _class-transformer_ and _class-validators_ dont automatically validate objects when passing them from service to service

Reply:
Not sure to what you are reffering specifically to, but we create `UpdateUserDto` based on received `CreateUserDto`, so no further validation is needed.